src/script/index.js

```diff
@@ -12,6 +12,10 @@ function error(err) {
 }
 window.navigator.geolocation.getCurrentPosition(success, error);
 
+// noticed a bug testing this around 11pm... current date was throwing an error in the fetch to skyscanner.  Not sure if related to this default
```

Comment:
interesting. it could be some problem in the API. 11pm CST is the next day in EST and it may be an error thinking its searching for a prior day if the API server is in EST+ time. I would suspect anyhow.